Dockerfile

```
@@ -0,0 +1,31 @@
+FROM python:3.9-alpine
+
+# install apps dependencies
+RUN apk update && apk add \
```

```suggestion
RUN apk add --no-cache \
```
consider reduce image size? https://stackoverflow.com/questions/49118579/alpine-dockerfile-advantages-of-no-cache-vs-rm-var-cache-apk